stress/common/src/main/java/alluxio/stress/master/MasterBatchTaskParameters.java

```diff
@@ -11,25 +11,43 @@
 
 package alluxio.stress.master;
 
-import alluxio.stress.common.BatchTaskParameters;
-
+import com.beust.jcommander.IParameterValidator;
 import com.beust.jcommander.Parameter;
+import com.beust.jcommander.ParameterException;
+import com.google.common.collect.ImmutableSet;
 
 /**
  * This holds all the parameters for Master Batch Task.
  */
-public class MasterBatchTaskParameters extends BatchTaskParameters {
-  @Parameter(names = {"--base"},
-      description = "The base directory path URI to perform operations in")
-  public String mBasePath = "alluxio:///stress-master-base";
+public class MasterBatchTaskParameters extends AbstractMasterBenchParameters {
+
+  @Parameter(description = "The batch task to perform. The current possible option is "
+      + "[MasterComprehensiveFileBatchTask]", validateWith = PossibleBatchTaskValidator.class)
+  public String mTaskName = "";
 
-  @Parameter(names = {"--num-files"}, description = "the number of files to operate on")
-  public int mNumFiles = 10000;
+  @Parameter(names = {"--cluster"},
+      description = "If true, runs the benchmark via the job service cluster. Otherwise, runs "
+          + "locally.")
+  public boolean mCluster = false;
 
-  @Parameter(names = {"--threads"}, description = "the number of concurrent threads to use")
-  public int mThreads = 256;
+  @Parameter(names = {"--in-process"},
+      description = "If true, runs the task in process. Otherwise, will spawn a new process to "
+          + "execute the task",
+      hidden = true)
+  public boolean mInProcess = false;
```

Comment:
Aren't these 2 from the `BaseParameters` class? Why do you need to define them here?